github/users_administration.go

```diff
@@ -38,13 +38,19 @@ func (s *UsersService) DemoteSiteAdmin(ctx context.Context, user string) (*Respo
 	return s.client.Do(ctx, req, nil)
 }
 
+// userSuspendOptions represents the reason a user is being suspended.
+type userSuspendOptions struct {
+	Reason string `json:"reason,omitempty"`
+}
+
 // Suspend a user on a GitHub Enterprise instance.
 //
 // GitHub API docs: https://developer.github.com/enterprise/v3/enterprise-admin/users/#suspend-a-user
-func (s *UsersService) Suspend(ctx context.Context, user string) (*Response, error) {
+func (s *UsersService) Suspend(ctx context.Context, user string, reason string) (*Response, error) {
```

Comment:
I'm thinking that we want to make this an `opt *UserSuspendOptions` and and allow it to be `nil` to continue the current behavior.

That way, it is more consistent with other endpoints in this package and it allows us to be a bit more flexible by not breaking this package's API in case GitHub wishes to add more options to this command.